metabase/metabase
# Add new data tooltip to row chart

Part of https://github.com/metabase/metabase/pull/27394 extracted to simplify the review process
[Product doc](https://www.notion.so/metabase/Offer-automated-percentages-on-stacked-charts-a8af42d2b81e458b827b679dac2b61c6)

## Changes

Adds the new chart tooltip alongside the old one. The new tooltip is only visible on the Row chart.

<img width="703" alt="Screen Shot 2023-01-04 at 8 08 53 PM" src="https://user-images.githubusercontent.com/14301985/210671263-ab76f5d3-d97d-4574-8d18-92f05229f54e.png">

## How to verify:

- Create a row chart with a breakout
- Hover it, ensure it shows breakout values with colors, percentages
- Toggle visibility of the breakouts, ensure it is reflected in the tooltip
- Create a row chart with multiple metrics
- Hover it, ensure it shows the hovered metric in the tooltip header
